drivers/scheduler/scheduler.go

```diff
@@ -161,6 +161,9 @@ type Driver interface {
 	// DeleteVolumes will delete all storage volumes for the given context
 	DeleteVolumes(*Context, *VolumeOptions) ([]*volume.Volume, error)
 
+	// GetPVCs returns all pvcs for the given context
+	GetPVCs(*Context) ([]*volume.PVC, error)
```

Comment:
Let's stick the Volume definition so we can keep the driver not directly tied with k8s. You should be able to use the existing `GetVolumes` with little or no modification required. If you need to add a phase or status, let's add that inside the Volume object.

Reply:
Both PVC and volume are just an object that wrap the information retrieved from k8s or dcos (not actually implemented). In k8s, PVC and volume are two different concepts, I am just afraid that combining them together in the GetVolumes can confuse people. Are you saying PVC is just a concept in k8s only? If that's the case, could we use a more generic name here?

Reply:
Yes, PVC is a k8s only concept. So to keep this generic, we have to use the Volume object.

Reply:
in k8s, pvc != volume, right? Just like in this test, the volume (pv) didn't get created, but pvc was created with pending phase. if getVolumes returns both PVCs and Volumes, wouldn't that be confusing?

Reply:
I suppose we could verify that GetVolumes() does not return any volumes? Then, we don't need GetPVCSs().

Reply:
Fixed by using GetVolumes().